### PR TITLE
Add a license

### PR DESCRIPTION
**Pull request by sjaks**

### CODE REVIEW

**Comment by Wiljami** [comment]
LETM :+1:

**Review by Wiljami — APPROVED** [review]

**Comment by sjaks** [comment]
@elguitar is there code you do not want licensed? If so, we could either:
- extend the license so that it doesn't affect your code
- or rethink adding a license

Removing your code from the repo is also an option, but your contributions to the repo are of utmost importance.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+                         The Pact in English
+
+1. You can copy, modify and distribute this piece of software.
+2. Any modifications of this code base MUST be distributed with the same license.
+3. You can use this software privately or for commercial purposes.
+4. This software is provided without warranty.
+5. The software author or license can not be held liable for anything.
+6. This piece of software is not a game and copies of it, classifiable as games, must not be published.
```
Comment (elguitar) [inline LICENSE:12]: This contradicts the first clause.

Also, "classifiable as games" is very debatable. Are we sure that the current version is not "classifiable as a game"?

Comment (sjaks) [reply LICENSE:12]: [In README.md on line 2](https://github.com/sjaks/demooni/blob/master/README.md#L2), it says

> This is not a game.

and also the definition of "game" is often considered to be

> a physical or mental competition conducted according to rules with the participants in direct opposition to each other

[Source](https://www.merriam-webster.com/dictionary/game?src=search-dict-hed)

This application doesn't have features that would make the user compete, *play a game*.

Do you have ideas how to improve or alter the license, @elguitar?

**Comment by sjaks** [comment]
Licensing should be re-considered. Closing this for now.

**Pull request closed without merging**